Firmware/MotorControl/gcode_input.c

```diff
@@ -0,0 +1,63 @@
+#include "grbl.h"
+#include "gcode_input.h"
+
+void consume_output() {
```

Comment:
Use (void) to denote zero function parameters in C.

Reply:
Done